code/modules/projectiles/projectile/bullets.dm

```diff
@@ -174,21 +186,12 @@
 	damage_type = TOX
 
 
-/obj/item/projectile/bullet/burstbullet//I think this one needs something for the on hit
+/obj/item/projectile/bullet/burstbullet
 	name = "exploding bullet"
-	damage = 20
 
-
-/obj/item/projectile/bullet/stunshot
-	name = "stunshot"
-	icon_state = "sshell"
-	damage = 5
-	stun = 10
-	weaken = 10
-	stutter = 10
-
-/obj/item/projectile/bullet/a762
-	damage = 25
+/obj/item/projectile/bullet/burstbullet/on_hit(var/atom/target, var/blocked = 0)
```

Comment:
Adds the on_hit for the exploding bullet. Not as good as the gyrojet rounds, but it's something.